public/index.html

```diff
@@ -19,13 +22,11 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>VotingWorks Ballot Marking Device</title>
+    <title>React App</title>
```

Comment:
couple small things lost like the title of the page, but not a big deal, we can fix later.

Reply:
pushed fix.